KP5-Topic2/KP4-Topic2/target/KP4-Topic2/WEB-INF/web.xml

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE web-app PUBLIC
```

Comment:
Please include the folder target in .gitignore file to avoid to upload to the repo these generated files